Brickwall/models.py

```diff
@@ -1,27 +1,98 @@
 from sqlalchemy import Column, Integer, String, DateTime, Boolean, ForeignKey, Float
+from flask_restplus import fields
 from . import db
 
 
 class Review(db.Model):
     __tablename__ = "review"
-    review_id = Column(Integer, primary_key=True, nullable=False)
+    id = Column(Integer, primary_key=True, nullable=False)
     member = Column(String(55), nullable=False, comment="user-name of reviewer")
-    type = Column(String(10), nullable=False, comment = 
+    type = Column(String(10), nullable=False, comment=
     """whether the person worked full time or co-op, takes a string, but precondition is it should take 
     one of two strings (\"full-time\" or \"co-op\")""")
-    company_id = Column(ForeignKey('company.company_id'), nullable=False, comment = \
+    company_id = Column(ForeignKey('company.id'), nullable=False, comment=
     "A reference to the company that the review is about")
 
-class Location:
+    def __init__(self, member, review_type, company_id):
+        """
+        :param member: the name of the CSH member
+        :param review_type: whether the person worked full time or co-op, takes a string, but precondition is it should take
+        one of two strings (\"full-time\" or \"co-op\"
+        :param company_id: The company that the review is associated with (can either be an id or an object)
+        """
+        self.member = member
+        self.type = review_type
+        self.company_id = company_id
+
+    @classmethod
+    def by_id(cls, id):
+        """
+        :param id:
+        :return: returning the object from the database based on its id
+        """
+        return cls.query.filter_by(id=id).first()
+
+    @classmethod
+    def get_model(cls, api):
+        """
+        :param api:
+        :return: returns the expected format of the input and output JSON
+        """
+        return api.model('Review', {
+            'id': fields.Integer(required=False),
+            'member': fields.String(required=True),
+            'type': fields.String(required=True),
+            'company_id': fields.Integer(required=True)
+        })
+
+
+    # Not sure if this is right
```

Comment:
Check out the [`vars()`](https://www.geeksforgeeks.org/vars-function-python/) builtin, pretty sure it does what you want.

Reply:
This also does work if `vars()` isn't working as you'd wish, looks fine to me